test/controllers/EvidenceDownloadController.test.ts

```diff
@@ -109,4 +114,46 @@ describe('EvidenceDownloadController', () => {
         }
 
     });
+
+    it('should render custom error page when the file service fails to download file with FileNotReady', async () => {
+
+        const app = createDefaultApp(fileTransferServiceProxy(Promise.reject(
+            new FileNotReadyError(`File download failed because "${FILE_ID}" file is either infected or has not been scanned yet`)),
+            metadataClean));
+
+        await request(app)
+            .get(EXPECTED_DOWNLOAD_LINK_URL)
+            .then(res => {
+                expect(res.status).to.equal(FORBIDDEN);
+                expect(res.text).to.contain(expectedDownloadErrorHeading);
+                expect(res.text).to.contain(expectedDownloadErrorMessage);
+            });
+    });
+
+    it('should render custom error page during download when the status is invalid', async () => {
+
+        const createMetadata = (status: undefined | null | string): FileMetadata => {
+            return {
+                av_status: status as any,
+                content_type: contentType,
+                id: FILE_ID,
+                name: 'hello.txt',
+                size: contentLength
+            };
+        };
+
+        for (const status of [undefined, null, 'infected', 'not-scanned']) {
```

Comment:
I think we can easily remove cases where status is undefined and null. These should never happen as per API contract.

Reply:
ok, i've removed the undefined/null. This test actually helped me identify a small bug I had introduced to the controller :D